src/applications/bmqbrkr/bmqbrkr.m.cpp

```diff
@@ -299,6 +299,8 @@ static int getConfig(bsl::ostream&      errorDescription,
     }
 
     taskEnv->d_config.appConfig().etcDir() = configDir;
+    taskEnv->d_config.appConfig().brokerVersion() =
+        bmqbrkrscm::Version::versionAsInt();
```

Comment:
So, before this, `brokerVersion` field was being populated during config decoding step at line 287 above?

Reply:
Yes.